applications/acdc/src/acdc_queue_listener.erl

```diff
@@ -339,7 +343,7 @@ handle_cast({'timeout_agent', RespJObj}, #state{queue_id=QueueId
     lager:debug("timing out winning agent"),
     send_agent_timeout(RespJObj, Call, QueueId),
     {'noreply', State#state{agent_id='undefined'}, 'hibernate'};
-handle_cast({'timeout_member_call'}, #state{delivery=Delivery
+handle_cast({'timeout_member_call', JObj}, #state{delivery=Delivery
```

Comment:
Please correct formatting on the #state{} record below this line. Should line back up with `delivery`